scripts/update-users.sh

```diff
@@ -17,6 +17,10 @@ add_user() {
     else
         echo "Adding user $user"
         useradd "$user" --create-home --shell /bin/bash
+        # delete and expire their password, forcing reset on first ssh login
+        passwd -de "$user"
+        # ensure home not world readable
+        chmod -R o-rwx "/home/$user"
```

Comment:
out of interest, the benchmark spec says you should set up monitoring to verify that users don't modify these permissions in the future (but that's definitely not something we could do effectively....)

Reply:
....this machine isn't going to have kissh on it, is it...?

Reply:
It's not, kissh is aimed at our public cloud infra, I think it needs to be separate.

But a kissh-like agent in the future perhaps.

Given we plan to have everyone with ssh access also have sudo, it's kinda moot. I just thought it was a good default that was easy to do.

Reply:
agree